src/ragas/experimental/testset/simulators/generate.py

```diff
@@ -71,7 +71,7 @@ def generate_with_langchain_docs(
 
         return self.generate(
             test_size=test_size,
-            scenarios=scenarios,
+            simulators=scenarios,
```

Comment:
Using simulators=scenarios created confusion on what is what. This is also present as an argument.